public/js/main.js

```diff
@@ -91,8 +90,9 @@ function renderToolbox() {
 }
 
 if (process.env.NODE_ENV !== "DEVTOOLS_PANEL") {
-  connectClient(tabs => {
+  connectClients().then((tabs) => {
     actions.newTabs(tabs);
+    renderNotConnected();
```

Comment:
Yup. I like the render toolbox suggestion. I'd like to clean that up too. Tablist is a bad name and we'll have a node UI at some point too. Thinking about calling it a toolbox and just expecting it to change over time